plugins/woocommerce-blocks/packages/components/text-input/test/validated-text-input.tsx

```diff
@@ -200,8 +206,8 @@ describe( 'ValidatedTextInput', () => {
 
 		await expect(
 			select( validationStore ).getValidationError( 'test-input' )
-		).not.toBe( undefined );
```

Comment:
This was passing because with the bug in `plugins/woocommerce-blocks/packages/components/text-input/validated-text-input.tsx`, the error was `{ message: "", hidden: true }`. There was no error to set simply because the field has no browser error to set (it has a value) and `customValidation` should be the be the setting the error, but it didn't.